zeppelin-web/src/components/navbar/navbar.html

```diff
@@ -43,9 +43,8 @@
             <li><a href="" data-toggle="modal" data-target="#noteNameModal"><i class="fa fa-plus"></i> Create new note</a></li>
             <li class="divider"></li>
             <div id="notebook-list" class="scrollbar-container">
-              <li class="filter-names" ng-include="'components/filterNoteNames/filter-note-names.html'"></li>
```

Comment:
We should make keep that line, and modify filter-note-names.html to have `ng-model="$parent.query"`